ddtrace/contrib/internal/openai/patch.py

```diff
@@ -163,7 +163,15 @@ def patched_client_init(openai, pin, func, instance, args, kwargs):
 
 
 def _traced_endpoint(endpoint_hook, integration, instance, pin, args, kwargs):
-    span = integration.trace(pin, endpoint_hook.OPERATION_ID)
+    client = getattr(instance, "_client", None)
+    base_url = getattr(client, "_base_url", None) if client else None
+
+    # only report LLM Obs spans if base_url has not been changed
+    span = integration.trace(
+        pin,
+        endpoint_hook.OPERATION_ID,
+        submit_to_llmobs=integration.is_default_base_url(str(base_url) if base_url else None),
```

Comment:
Can we move the `submit_to_llmobs` logic to inside `OpenAIIntegration.trace()` instead? Feels like it will be simpler to just pass in the `base_url` and work with that inside